tests/cypress/integration/actions_users/case_73_reset_password_notification.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+context('Reset password notification.', () => {
+    const caseId = '73';
+    const dummyEmail = 'admin@local.local'
```

Comment:
Why not use email from `cypress.json`?

Reply:
Please take a look for my comment above. I got an error.

Reply:
Sure, didn't notice your comment.